Dockerfile.dev

```diff
@@ -0,0 +1,7 @@
+FROM python:3.10-slim-bullseye
```

Comment:
Deze container heeft toch geen Python nodig? Kunnen we `nginx:1.23.3-alpine-slim` gebruiken?

Reply:
true, was pure luiheid